torchao/dtypes/fbgemm_int4_tensor.py

```diff
@@ -32,20 +32,44 @@
 
 
 class FbgemmInt4Tensor(TorchAOBaseTensor):
+    """
+    Groupwise int4 weight only quantization
+
+    Tensor Attributes:
+        packed_weight: packed int4 weight, either 2D (N, K/2) or 3D (B, N, K/2), last dimension is packed
```

Comment:
when is a weight a 3D tensor, and why is the batch dimension in here? could you share a specific example